ams/server/src/main/java/com/netease/arctic/server/table/TableRuntime.java

```diff
@@ -408,4 +420,148 @@ public long getQuotaTime() {
   public double calculateQuotaOccupy() {
     return getQuotaTime() / tableConfiguration.getOptimizingConfig().getTargetQuota();
   }
+
+  /**
+   * Get all valid blockers.
+   *
+   * @return all valid blockers
+   */
+  public List<TableBlocker> getBlockers() {
+    blockerLock.lock();
+    try {
+      return getAs(TableBlockerMapper.class,
+          mapper -> mapper.selectBlockers(tableIdentifier, System.currentTimeMillis()));
+    } catch (Exception e) {
+      LOG.error("failed to get blockers for {}", tableIdentifier, e);
```

Comment:
Why do we need to print error logs here even though the exception has been thrown?

Reply:
Ok, I changed to print the error logs in `TableManagementService`.